templates/ecr/content/ecr.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_ecr_repository" "${{ values.product_name }}_ecr" {
+  name                 = "${{values.product_name }}_ecr"
+  image_tag_mutability = "MUTABLE"
+
+  image_scanning_configuration {
+    scan_on_push = true
+  }
+
+  tags = {
+    CostCentre = ${{ values.billing_code | dump }}
+    Terraform  = true
+  }
+}
+
+resource "aws_ecr_lifecycle_policy" "${{ values.product_name }}_exire_untagged" {
```

Comment:
I think the `_exire_untagged` meant to be `_expire_untagged`?

Reply:
Sorry! Good catch - I have fixed it.